### DISCUSSION TITLE
Fetching artifact metadata failed. No artifacts named "github-pages" were found for this workflow run.

### DISCUSSION THREAD

**Discussion by douagz** [community/community#84008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After **deploy-pages@v4** and **upload-pages-artifact@v3**, there's an error, it is similiar with the issue 285/288, which is fixed in version 4.0.1. I updated the file to use **deploy-pages@v4.0.1** and added actions: read permission, but the error still occurred.
How can i solve this problem?
![image](https://github.com/community/community/assets/122457822/4a903f5c-00fc-46ab-a027-11bd556e6374)

**Comment by douagz** [comment]
I just updated to deploy-pages@v3 and upload-pages-artifact@v2, the error didn't occur. 

What version do you use to depoly gh-pages?

**Comment by cnehm006** [comment]
It's great that you've tried updating to the latest versions and adjusting permissions, but it's puzzling that the error persists.

Here are a few steps you might consider to resolve this:

- Compatibility Check: Ensure that deploy-pages@v4.0.1 and upload-pages-artifact@v3 are compatible with each other and with your current workflow setup. Sometimes, specific versions of actions may not work well together.

- Review Action's Release Notes: Go through the release notes for deploy-pages@v4.0.1. There might be specific changes or additional configuration requirements introduced in this version that are necessary for your workflow.

- Examine Action Permissions: You've added read permissions, which is a good start. However, depending on the specifics of your workflow, additional permissions might be required. Verify that the permissions set in your workflow are sufficient for the tasks being performed.

- Workflow File Validation: Double-check your YAML file for any potential errors or misconfigurations. An incorrect setting or a syntax error can sometimes cause unexpected behavior.

Since reverting to deploy-pages@v3 and upload-pages-artifact@v2 works without errors, you might continue using these versions temporarily. This approach is practical if those versions meet your project's requirements and don't pose any security or functionality issues.

Hope this helps!

**Comment by douagz** [comment]
@cnehm006 Many thanks for your suggestions! Yes, deploy-pages@v3 and upload-pages-artifact@v2 works without errors, so I use them in my workflow now.
